extensions/rich_text_components/Link/Link.html

```diff
@@ -3,6 +3,6 @@
   being added before and after the link. See http://stackoverflow.com/q/5078239
 -->
 <script type="text/ng-template" id="richTextComponent/Link"><!--
-  --><a ng-if="showUrlInTooltip" href="<[url]>" target="<[target]>" tooltip="<[url]>"><[text]></a><!--
-  --><a ng-if="!showUrlInTooltip" href="<[url]>" target="<[target]>"><[url]></a><!--
+  --><a ng-if="showUrlInTooltip" href="<[url]>" target="<[target]>" tooltip="<[url]>" tabIndex="<[tabIndexVal]>"><[text]></a><!--
```

Comment:
Yeah well I just realized it does not matter if it all lowercase or not. 
I will fix it.